wp-content/themes/eighshift-task/src/rest/fields/class-posts-fields.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * REST Field on /posts/ route
+ *
+ * @since 1.0.0
+ * @package Eighshift_Task\Rest\Fields
+ */
+
+namespace Eighshift_Task\Rest\Fields;
+
+use Eightshift_Libs\Core\Service;
+use Eightshift_Libs\Routes\Registrable_Field;
+
+/**
+ * Class for registering and changine REST fields for /posts/ route.
+ */
+class Posts_Fields implements Service, Registrable_Field {
```

Comment:
rest field should have callback interface on it

Reply:
👍

Reply:
👍